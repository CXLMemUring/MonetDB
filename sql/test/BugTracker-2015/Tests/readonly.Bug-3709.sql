create table s10 (x int); 
start transaction; 
insert into s10 values (8); 
alter table s10 set read only; 
explain select * from s10; 
rollback;
drop table s10; 
