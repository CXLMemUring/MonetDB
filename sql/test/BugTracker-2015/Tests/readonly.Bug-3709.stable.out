stdout of test 'readonly.Bug-3709` in directory 'sql/test/BugTracker-2015` itself:


# 21:17:24 >  
# 21:17:24 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31025" "--set" "mapi_usock=/var/tmp/mtest-5516/.s.monetdb.31025" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/home/niels/scratch/rc-clean/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2015" "--set" "mal_listing=0" "--set" "embedded_r=yes"
# 21:17:24 >  

# MonetDB 5 server v11.19.10
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2015', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 7.333 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:31025/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-5516/.s.monetdb.31025
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# MonetDB/R   module loaded


# 21:17:25 >  
# 21:17:25 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-5516" "--port=31025"
# 21:17:25 >  

#create table s10 (x int); 
#start transaction; 
#insert into s10 values (8); 
[ 1	]
#alter table s10 set read only; 
#trace select * from s10; 
% sys.s10 # table_name
% x # name
% int # type
% 1 # length
[ 8	]
#trace select * from s10; 
% .trace,	.trace # table_name
% usec,	statement # name
% bigint,	clob # type
% 3,	134 # length
[ 9,	"X_1=0@0:void := querylog.define(\"trace select * from s10;\":str, \"default_pipe\":str, 13:int);"	]
[ 11,	"X_4=0:int := sql.mvc();"	]
[ 61,	"X_15=[1]:bat[:str] := bat.pack(\"x\":str);"	]
[ 63,	"X_16=[1]:bat[:str] := bat.pack(\"int\":str);"	]
[ 59,	"X_18=[1]:bat[:int] := bat.pack(0:int);"	]
[ 55,	"X_17=[1]:bat[:int] := bat.pack(32:int);"	]
[ 109,	"X_14=[1]:bat[:str] := bat.pack(\"sys.s10\":str);"	]
[ 323,	"C_5=[1]:bat[:oid] := sql.tid(X_4=0:int, \"sys\":str, \"s10\":str);"	]
[ 33,	"X_11=[1]:bat[:int] := sql.bind(X_4=0:int, \"sys\":str, \"s10\":str, \"x\":str, 0:int);"	]
[ 24,	"X_12=[1]:bat[:int] := algebra.projection(C_5=[1]:bat[:oid], X_11=[1]:bat[:int]);"	]
[ 848,	"barrier X_70=false:bit := language.dataflow();"	]
[ 62,	"sql.resultSet(X_14=[1]:bat[:str], X_15=[1]:bat[:str], X_16=[1]:bat[:str], X_17=[1]:bat[:int], X_18=[1]:bat[:int], X_12=[1]:bat[:int]);"	]
#rollback;
#drop table s10; 

# 21:17:25 >  
# 21:17:25 >  "Done."
# 21:17:25 >  

